.repo-metadata.json

```diff
@@ -3,11 +3,12 @@
     "name_pretty": "Google Auth OAuthlib",
     "client_documentation": "https://googleapis.dev/python/google-auth-oauthlib/latest",
     "issue_tracker": "https://github.com/googleapis/google-auth-library-python-oauthlib/issues",
-    "release_level": "alpha",
+    "release_level": "preview",
     "language": "python",
     "library_type": "AUTH",
     "repo": "googleapis/google-auth-library-python-oauthlib",
     "distribution_name": "google-auth-oauthlib",
     "default_version": "",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "google-auth-oauthlib"
```

Comment:
This is not a GAPIC library, is an `api_shortname` required?

Reply:
Fixed